verify.js

```diff
@@ -1,21 +1,25 @@
 const debug = require('diagnostics')('warehouse.ai-api-client:verify');
-const request = require('request');
+const https = require('https');
+const nodeFetch = require('node-fetch');
+const fetch = require('fetch-retry')(nodeFetch);
```

Comment:
where's the configuration for the retries? just using all the defaults? Should it be overrideable by the client via various retry options?